packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponent.tsx

```diff
@@ -128,7 +120,15 @@ export const ChatMyMessageComponent = (props: ChatMyMessageComponentProps): JSX.
         onSubmit={async (text, metadata, options) => {
           props.onUpdateMessage &&
             message.messageId &&
-            (await props.onUpdateMessage(message.messageId, text, metadata, options));
+            (await props.onUpdateMessage(
+              message.messageId,
+              text,
+              /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
+              {
+                metadata: metadata ?? {},
```

Comment:
As `onUpdateMessage` func takes optionals, I'm wondering if we need to add empty object and array for these props